com.microsoft.java.debug.core/src/main/java/com/microsoft/java/debug/core/adapter/handler/StepRequestHandler.java

```diff
@@ -438,15 +439,19 @@ private boolean shouldDoExtraStepInto(int originalStackDepth, Location originalL
         return true;
     }
 
-    private boolean isSameLocation(Location original, Location current) {
+    private boolean isSameLocation(Location original, Location current, MethodInvocation targetStepIn) {
```

Comment:
Look at the caller, actually we took the first parameter as `current`, the second one as `original`. So we should change the signature definition to `private boolean isSameLocation(Location current, Location original, MethodInvocation targetStepIn)`.